CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing To A Rearc AWS Data Exchange Product
```

Comment:
suggestion: `Contributing to a Rearc AWS Data Exchange product`

Reply:
I've adjusted the title per your suggestion, and made a few other minor edits